localdev/test.sh

```diff
@@ -118,6 +123,24 @@ fileShouldExist "not_so_spacey_target/space content.txt"
 fileShouldExist "music/spacey test.mp3"
 
 fileShouldExistWithContent "mappedDuringStart/some.txt" "some content"
+
+actCTime=$(stat -f '%c' mappedDuringStart/some.txt)
+now=$(date +%s)
+earliestAllowedCTime=$((now - 10)) # give 10 seconds, to allow between the copying and when this is evaluated
+
+
+# checking for a range and that it is not equal to the ctime before the container was started. Because this is tested on
```

Comment:
i'd put this test after the below assertion `actCTime != fileCTime`


and add a comment like

`# the creation time is set to the time the file has been copied`

try to get the ts when the container has been started and the initial copy script is invoked. then you can compare to `containerStartedTs-1sec` instead of `now-10`